e2e2/test/cases/neuron/manifests/single_node_test_neuronx.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: neuronx-single-node
+  labels:
+    app: neuronx-single-node
+spec:
+  template:
+    metadata:
+      labels:
+        app: neuronx-single-node
+    spec:
+      containers:
+      - name: neuronx-single-node-test
+        image: {{.NeuronSingleNodeTestImage}}
```

Comment:
we should be able to use the same image for single and multi-node tests, right? lets use a more generic name for this template var

Reply:
Change to the `NeuronTestImage`